evap/grades/views.py

```diff
@@ -134,7 +134,7 @@ def toggle_no_grades(request):
 @grade_downloader_required
 def download_grades(request, grade_document_id):
     grade_document = get_object_or_404(GradeDocument, id=grade_document_id)
-    if not grade_document.course.grades_activated:
+    if not grade_document.course.grades_activated and not request.user.is_grade_publisher:
         return HttpResponseForbidden()
```

Comment:
as per our discussion, this commit is not the right solution anymore, or is it?

Reply:
we didn't decide on anything yet, so there currently is no solution.

Reply:
i will probably remove this commit from the pr